components/supervisor/pkg/supervisor/services.go

```diff
@@ -673,55 +663,22 @@ func (is *InfoService) RegisterREST(mux *runtime.ServeMux, grpcEndpoint string)
 	return api.RegisterInfoServiceHandlerFromEndpoint(context.Background(), mux, grpcEndpoint, []grpc.DialOption{grpc.WithTransportCredentials(insecure.NewCredentials())})
 }
 
-func (is *InfoService) getDefaultWorkspaceImage(ctx context.Context) string {
-	is.cacheMu.RLock()
-
-	if time.Since(is.cacheTimestamp) < infoCacheDuration {
-		is.cacheMu.RUnlock()
-		return is.defaultWorkspaceImage
-	}
-	is.cacheMu.RUnlock()
-
-	is.cacheMu.Lock()
-	defer func() {
-		is.cacheTimestamp = time.Now()
-		is.cacheMu.Unlock()
-	}()
-	if time.Since(is.cacheTimestamp) < infoCacheDuration {
-		return is.defaultWorkspaceImage
-	}
-
-	is.defaultWorkspaceImage = is.cfg.DefaultWorkspaceImage
-	if is.GitpodService == nil {
-		return is.defaultWorkspaceImage
-	}
-	wsImage, err := is.GitpodService.GetDefaultWorkspaceImage(ctx)
-	if err == nil {
-		is.defaultWorkspaceImage = wsImage
-	} else {
-		log.WithError(err).Error("failed to get default workspace image")
-	}
-	return is.defaultWorkspaceImage
-}
-
 // WorkspaceInfo provides information about the workspace.
 func (is *InfoService) WorkspaceInfo(ctx context.Context, req *api.WorkspaceInfoRequest) (*api.WorkspaceInfoResponse, error) {
```

Comment:
Maybe make sense to add a comment here for future us that this function is performance critical for initial opening.